cli/src/klio_cli/commands/job/gke.py

```diff
@@ -184,11 +184,11 @@ def _apply_deployment(self):
                 body=dep, namespace=namespace
             )
             deployment_name = resp.metadata.name
-            current_cluster = self.kubernetes_active_context["name"][-1]
+            current_cluster = self.kubernetes_active_context["name"]
```

Comment:
It looks like you undid the change in the previous commit - was this intentional? (similarly on line 252)

Reply:
yeah it was intentional. `self.kubernetes_active_context["name"][-1]` comes back empty for me vs `self.kubernetes_active_context["name"]` properly has the current cluster. Asking for some crowd sourcing to see if anyone can reproduce the need for the `[-1]` since users can only have one active context at a time.

Reply:
ahhh gotcha. Here's a makeshift (non)repro:

```pycon
>>> from kubernetes import config as k8s_config
>>> _, active_context = k8s_config.list_kube_config_contexts()
>>> active_context["name"]
'gke_gke-xpn-1_europe-west1_testing-europe-west1'
```

Not sure where the indexing into the last element comes from either. I guess if you want to play it safe, you can do something like:

```py
current_cluster = self.kubernetes_active_context["name"]
if not isinstance(current_cluster, str):
  raise SomeException("can't detect current cluster name...blahblahblah")
```
or if we want to trust that, if it is a list, we should take the last element:

```py
current_cluster = self.kubernetes_active_context["name"]
if isinstance(current_cluster, list):
  current_cluster = current_cluster[-1]
```

Or we could not be defensive and just have it explode ¯\\_(ツ)_/¯ unless @fallonchen remembers how she got a list instead of a string. Without diving much into where [this gets set](https://github.com/kubernetes-client/python-base/blob/6b0104ffb9dd2f96d47d075ea3d30f69ea124ce4/config/kube_config.py#L598), it does look like it should be singular.

Reply:
I'm not able to reproduce it either, which makes me think this was copy-pasta and that I was using an older version of my code when I tested it. I think it's safe to remove.  Sorry about the mess/confusion 😬 .